sdk/tables/azure-data-tables/samples/async_samples/sample_create_delete_table_async.py

```diff
@@ -95,15 +95,15 @@ async def create_from_table_client(self):
 
     async def delete_from_table_client(self):
         from azure.data.tables.aio import TableClient
-        from azure.core.exceptions import ResourceNotFoundError
+        from azure.core.exceptions import HttpResponseError
 
         # [START delete_from_table_client]
         async with TableClient.from_connection_string(conn_str=self.connection_string, table_name=self.table_name) as table_client:
             try:
                 await table_client.delete_table()
                 print("Deleted table {}!".format(self.table_name))
-            except ResourceNotFoundError:
-                print("Table could not be found")
+            except HttpResponseError:
```

Comment:
Wondering if we should just remove these exception wrappers from the sample... 
We technically wouldn't have been catching these before.

Reply:
I agree, I removed them.

Reply:
This one didn't get removed ;)